nideep/datasets/washington-rgbd/washington_rgbd.py

```diff
@@ -0,0 +1,270 @@
+import os
+import numpy as np
+import cv2
+import argparse
+import shutil
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+
+class WashingtonRGB:
+    ROOT_DEFAULT = '/mnt/raid/data/ni/dnn/pduy/rgbd-dataset'
+    CSV_DEFAULT = '/mnt/raid/data/ni/dnn/pduy/rgbd-dataset/rgbd-dataset.csv'
+    CSV_INTERPOLATED_DEFAULT = '/mnt/raid/data/ni/dnn/pduy/rgbd-dataset/rgbd-dataset-interpolated.csv'
+
+    def __init__(self, root_dir='', output=''):
+        self.root_dir = root_dir if root_dir != '' else self.ROOT_DEFAULT
+        self.output = output
+
+    # Load the dataset metadata to a Pandas dataframe and save the result to a csv file 
+    # if it did not exists
+    # The missing pose values will be saved as -1
+    def load_metadata(self, csv_output=''):
+        if os.path.exists(self.CSV_DEFAULT):
+            print 'reading from ' + self.CSV_DEFAULT
```

Comment:
use logging instead of print. See [example](https://github.com/nigroup/nideep/blob/master/nideep/datasets/amfed/amfed.py#L112)